src/components/fields/derived.js

```diff
@@ -17,22 +17,14 @@ export const AxisAnchorDropdown = connectToContainer(UnconnectedDropdown, {
     const {localize: _} = context;
     let options = [];
 
-    if (
-      plotProps.fullContainer &&
-      plotProps.fullContainer._subplot &&
-      plotProps.fullContainer._subplot.includes('xaxis')
-    ) {
+    if (props.attr.startsWith('x')) {
```

Comment:
this seems a little fragile, is there a way to make it more precise?
what if there's other axes related attributes that start with x or y..

Reply:
good call, i'll make it more precise
but generally, this guys get attr supplied to them